app/services/backend_service.rb

```diff
@@ -0,0 +1,25 @@
+class BackendService
+  def self.conn
+    Faraday.new(url: ENV["BACKEND_URL"])
+  end
+
+  def self.flights_search(query)
+    response = conn.get('/flight_search') do |req|
+      req.params[:fly_from] = query[:fly_from]
+      req.params[:date_to] = query[:date_to]
+      req.params[:date_from] = query[:date_from]
+      req.params[:nights_in_dst_to] = query[:nights_in_dst_to]
+      req.params[:nights_in_dst_from] = query[:nights_in_dst_from]
+      req.params[:flight_type] = 'round'
+      req.params[:one_for_city] = 1
+      req.params[:partner_market] = 'us'
+      req.params[:curr] = 'USD'
+      req.params[:sort] = 'price'
+      req.params[:ret_from_diff_airport] = 0
+      req.params[:partner] = 'picky'
```

Comment:
Jesse and I refactored the Flight-API endpoint last night with these default parameters that will be present in all of our calls. I don't think it should cause any problems to have them defined in both the BE and Flight-API repos, but we should be able to remove lines 13-19 here, as we set them in the microservice.

Reply:
Great! We took these out. Thank you!